gen3_util/config/config.py

```diff
@@ -10,7 +11,11 @@
 
 def default():
     """Load config from installed package."""
-    return Config(**yaml.safe_load(pkg_resources.open_text(gen3_util, 'config.yaml').read()))
+    if sys.version_info[:3] <= (3, 9):
+        return Config(**yaml.safe_load(pkg_resources.open_text(gen3_util, 'config.yaml').read()))
```

Comment:
I'm not familiar with the `sys.version_info[:3] <= (3, 9)` and `**yaml` syntax — is this just checking package versions and installing the latest ones?

Reply:
`**`: aka turn a dictionary into keyword arguments
sys.version_info[:3] <= (3, 9) : check python version